enhancements/etcd/assisted-install.md

```diff
@@ -0,0 +1,221 @@
+---
+title: support-assisted-installer
+
+authors:
+  - "@hexfusion"
+reviewers:
+  - "@deads2k"
+  - "@skolicha"
+  - "@dmace"
+approvers:
+  - TBD
+creation-date: 2020-09-16
+last-updated: 2020-09-16
+status: provisional
+---
+
+# backup config controller
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+The assisted installer was designed to utilize three bare-metal machines to
+provision the control-plane. In order to do this after bootstrap is complete
+the bootstrap node is repurposed into the third master node.
+
+## Challenges
+The installer currently does not provide an `install_type` key for the
+install-config. So no direct signal is passed from the installer to operator of
+the exact intent to perform a non standard install process.
+
+The result is the operator must make educated assumptions as to the install intent based
+on observations from the cluster itself.
+
+## Motivation
+Currently the `cluster-etcd-operator` does not tolerate less than 3 master
+nodes during install time.
+
+Reasons outlined for cluster with less than 3 master nodes as unsafe non HA.
+1. Violates operational best practices for etcd - unstable
+1. Allows non-HA configurations which we cannot support in production - unsafe and non-HA
+1. Allows a situation where we can get stuck unable to re-achieve quorum, resulting in cluster-death - unsafe, non-HA, non-production, unstable
+1. The combination of all these things makes the situation unsupportable. 
+
+To allow toleration of this check we provided an unsupportedConfigOverrides
+`useUnsupportedUnsafeNonHANonProductionUnstableEtcd`. As assisted installer is
+now moving to GA a supported toleration is required.
+
+## Goals
+
+1.  Provide a supported solution for assisted installer that will allow
+    `cluster-etcd-operator` to tolerate 2 master replicas as install complete.
+
+## Non-Goals
+
+1.  Support the current unsupportedConfigOverrides
+1.  Add support for less than 3 master node clusters
+
+## Currently Supported Functionality 
+Currently `cluster-etcd-operator` and OpenShift only supports clusters with
+three master nodes as install complete.
```

Comment:
It does not change in the assisted install flow. Once installation is completed, there are 3 etcd masters.
We finish all the pivot way before ocp installation is completed.

Reply:
Right but the point is within the context of supported means. I don't disagree in general.